### PR TITLE
fix: ignore generating non-diregapic pagination with map response

### PR DESCRIPTION
**Pull request by summer-ji-eng**
Ignore the non diregapic pagination methtod where the response contain maps.
```
message PagedExpandLegacyMappedResponse {
  // The words that were expanded, indexed by their initial character.
  // (-- aip.dev/not-precedent: This is a legacy, non-standard pattern that violates
  //     aip.dev/158. Ordinarily, this should be a `repeated` field, as in PagedExpandResponse. --)
  map<string, PagedExpandResponseList> alphabetized = 1;

  // The next page token.
  string next_page_token = 2;
}
```
See `PagedExpandLegacyMapped` in [showcase proto](https://github.com/googleapis/gapic-showcase/blob/master/schema/google/showcase/v1beta1/echo.proto#L101-L106)

### CODE REVIEW

File: baselines/compute/test/gapic_addresses_v1.ts.baseline [hunk 1]
```diff
@@ -343,7 +343,6 @@ describe('v1.AddressesClient', () => {
                 .getCall(0).calledWith(request, expectedOptions, undefined));
         });
     });
-
```
Comment (alexander-fenster) [inline baselines/compute/test/gapic_addresses_v1.ts.baseline:346]: Maybe it's better to put this empty line back if not too hard?

File: typescript/src/schema/proto.ts [hunk 2]
```diff
@@ -316,6 +317,24 @@ function pagingResponseType(
   return '.google.protobuf.FieldDescriptorProto.Type.' + type;
 }
 
+// Ignore non-diregapic pagation method where its response type contains a map.
+function ignorePagingMethod(
```
Comment (alexander-fenster) [inline typescript/src/schema/proto.ts:321]: Let's rename it to `ignoreMapPagingMethod` here and everywhere, otherwise the name is confusing.

**Review by alexander-fenster — APPROVED** [review]

File: baselines/compute/test/gapic_addresses_v1.ts.baseline (hunk 1 above)
Comment (summer-ji-eng) [reply baselines/compute/test/gapic_addresses_v1.ts.baseline:346]: Agree.

File: typescript/src/schema/proto.ts (hunk 2 above)
Comment (summer-ji-eng) [reply typescript/src/schema/proto.ts:321]: agree

**Pull request merged**